src/Directory.Build.props

```diff
@@ -0,0 +1,41 @@
+<Project>
+
+  <PropertyGroup>
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+
+    <LangVersion>latest</LangVersion>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <CurrentYear>$([System.DateTime]::Now.ToString(yyyy))</CurrentYear>
```

Comment:
move down one line

Reply:
Adressed in https://github.com/destructurama/fsharp/commit/cd5adf371f201c6de509bd680c1efdbf99bf0e81

Reply:
Nice - all of these are definitely drive by random commentary - entirely up to you want to do.not do as the key thing is that you've done a rgeat job already